xml/article_virtualization.xml

```diff
@@ -24,20 +24,20 @@
 
    <para>
     To see the degree of interference from running within a particular &kvm;
-    configuration versus running on bare-metal configuration, Each RT
+    configuration versus running on bare-metal configuration, each RT
     application has to be assessed individually. We do not give any specific
     guarantees on performance or deadlines been missed.
    </para>
 
    <para>
-    Virtualization inevitably introduces overhead but there is currently no
+    Virtualization inevitably introduces overhead, but there is currently no
     rule of thumb for the performance penalty incurred. It is up to each RT
     application developer to set performance and deadline requirements and
     evaluate if those requirements are met.
    </para>
 
    <para>
-    This guide provides the following 3 examples for user reference only.
+    This guide provides the following three examples for user reference only.
```

Comment:
I would just remove the number altogether to enable later amendments without having to increment some number here. Also not sure what "user reference" rather than "reference" brings to the table.

```suggestion
    This guide provides the following examples for reference only.
```